internal/rbd/replicationcontrollerserver.go

```diff
@@ -834,6 +835,116 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 	return resp, nil
 }
 
+// GetVolumeReplicationInfo extracts the RBD volume information from the volumeID, If the
+// image is present, mirroring is enabled and the image is in primary state.
+// If yes it will get the volume replication information.
+func (rs *ReplicationServer) GetVolumeReplicationInfo(ctx context.Context,
+	req *replication.GetVolumeReplicationInfoRequest,
```

Comment:
move all parameters on a new line, it is a recommendation in our own coding standard

Reply:
just followed the format of other operations defined above and all seems to be this way.